### PR TITLE
[DOC-2145] update docs

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,6 +190,8 @@ file, if necessary.
 For detailed information on how to create models that work in these environments, reference the links above for each environment.
 
 ## Building your own environment
+>Note: This is not common that you want to do this. For example: you don't want to use drum and want to implement your own prediction server
```
Comment (nate-goudreault) [inline README.md:193]:
```suggestion
>Note: DataRobot recommends using an environment template and not building your own environment except for specific use cases. (For example: you don't want to use **drum** but you want to implement your own prediction server.)
```

Comment (yakov-g) [reply README.md]: fixed

**Comment by yakov-g** [comment]
jarvis test cmrunner_integration

**Comment by yakov-g** [comment]
jarvis test functional_drop_in

**Comment by yakov-g** [comment]
jarvis test functional_drop_in

**Comment by datarobot-jarvis** [comment]
:dollar: Total cost of executing tests in Jenkins: $**3.24** USD.

[What is this all about?](http://bit.ly/2BahK4V)

**Pull request merged**